server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -662,18 +672,50 @@ const orderMutations = {
         if (!req.remoteUser.isAdminOfCollective(fromAccount)) {
           throw new Unauthorized();
         }
-      }
 
-      const toAccount = await fetchAccountWithReference(paymentIntentInput.toAccount, { throwIfMissing: true });
-      const hostStripeAccount = await toAccount.getHostStripeAccount();
-      const host = await toAccount.getHostCollective();
+        let pm = await models.PaymentMethod.findOne({
+          where: {
+            CollectiveId: fromAccount.id,
+            service: PAYMENT_METHOD_SERVICE.STRIPE,
+            type: PAYMENT_METHOD_TYPE.PAYMENT_INTENT,
+            data: {
+              stripeAccount: hostStripeAccount.username,
+            },
+          },
+        });
+
+        const customer = await stripe.customers.create(
+          {
+            email: req.remoteUser.email,
+            description: `${config.host.website}/${fromAccount.slug}`,
+          },
+          !isPlatformHost
+            ? {
+                stripeAccount: hostStripeAccount.username,
+              }
+            : undefined,
+        );
```

Comment:
Will this return the existing customer if the email was already created, or are we creating one disposable customer per intent?

Reply:
Neither. This stripe call will always create a new user. But we are only creating a user once per stripe host account, if the user was created before we have it stored in the payment method.

Reply:
Not really. `stripe.customers.create` is being called outside the conditional `if (!pm)`.

Reply:
Ops, fixed that!

Reply:
I refactored to store the stripe customer id as a connected account instead of a dummy payment method.